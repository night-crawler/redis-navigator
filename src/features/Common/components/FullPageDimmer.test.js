/* eslint-disable react/react-in-jsx-scope,no-undef */
import { FullPageDimmer } from '.';

describe('FullPageDimmer', () => {
    it('should render FullPageDimmer', () => {
        const wrapper = shallow(
            <FullPageDimmer message='lol' />
        );
        expect(wrapper).toMatchSnapshot();
    });
});
