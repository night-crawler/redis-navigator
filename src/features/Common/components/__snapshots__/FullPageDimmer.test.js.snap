// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullPageDimmer should render FullPageDimmer 1`] = `
<div
  style={
    Object {
      "height": "100%",
    }
  }
>
  <Dimmer
    active={true}
    inverted={true}
  >
    <Loader
      size="massive"
    >
      lol
    </Loader>
  </Dimmer>
</div>
`;
